### DISCUSSION TITLE
5.5.0-2

### DISCUSSION THREAD

**Discussion by electronstudio** [electronstudio/jaylib#69] [Announcements]
* adds binaries for 32bit Windows x86 (with OpenGL 2.1, not the usual 3.3, because I'm not confident these old machines can support it, but do let me know.)

<hr /><em>This discussion was created from the release <a href='https://github.com/electronstudio/jaylib/releases/tag/v5.5.0-2'>5.5.0-2</a>.</em>

**Comment by Luigi654** [comment]
![imagen](https://github.com/user-attachments/assets/61e31038-631b-4056-a6c3-c86957d6496b)
works nice the kubuntu 20.10 x64 with virtual machine (at least)

**Reply by Luigi654** [reply]
[spec_test.txt](https://github.com/user-attachments/files/17999052/spec_test.txt)
![imagen](https://github.com/user-attachments/assets/0a3a902d-2e91-4587-adc0-adafe39a44c4)
working more on win 7 32 bits with a virtual machine, more details in the text, looks like as lighting of quake 3 engine (the low fps for my pc, not the VM)

**Comment by Luigi654** [comment]
can add array as part of Pointer?
